src/libsync/configfile.cpp

```diff
@@ -850,8 +850,9 @@ void ConfigFile::setDownloadLimit(int kbytes)
 QPair<bool, qint64> ConfigFile::newBigFolderSizeLimit() const
 {
     auto defaultValue = Theme::instance()->newBigFolderSizeLimit();
-    qint64 value = getValue(newBigFolderSizeLimitC, QString(), defaultValue).toLongLong();
-    bool use = value >= 0 && getValue(useNewBigFolderSizeLimitC, QString(), true).toBool();
+    const auto fallback = getValue(newBigFolderSizeLimitC, QString(), defaultValue).toLongLong();
+    const auto value = getPolicySetting(QLatin1String(newBigFolderSizeLimitC), fallback).toInt();
```

Comment:
Shouldn't that be `toLongLong()` here as well? Otherwise `fallback` and `value` have different types which feels odd (and in the case of value I guess it could overflow easily).

Reply:
Yes thats right. Thank you for your review.